ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Element.java

```diff
@@ -0,0 +1,140 @@
+package com.spectralogic.ds3autogen.c.models;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3autogen.api.models.Ds3Annotation;
+import com.spectralogic.ds3autogen.utils.Helper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.ParseException;
+
+
+public class Element {
+    private static final Logger LOG = LoggerFactory.getLogger(Element.class);
+
+    private final String name;
+    private final String type;
+    private final String componentType;
+    private final ImmutableList<Ds3Annotation> annotations;
+
+    public Element(
```

Comment:
These functions were in the "CHelper" class, but the behavior is tied to a specific Element or Type, so I moved it here.  Should they go back to the CHelper, or create TypeGenerator etc that lives at the same level as the CCodeGenerator?